Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -142,7 +142,7 @@ open class ChatChannelVC:
 
     open func chatMessageListVC(_ vc: ChatMessageListVC, messageAt indexPath: IndexPath) -> ChatMessage? {
         guard indexPath.item < channelController.messages.count else { return nil }
-        return channelController.messages[indexPath.item]
+        return channelController.messages[safe: indexPath.item]
```

Comment:
You can remove the line above since in this case, we were actually protecting against an invalid index 👍

Reply:
@polqf furtherly, the `return` can be removed from the single line expression.